packages/core/contracts-ovm/insured-bridge/implementation/OptimismMessenger.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../external/OVM_CrossDomainEnabled.sol";
+
+/**
+ * @notice Sends cross chain messages Optimism L2 network.
+ */
+contract OptimismMessenger is OVM_CrossDomainEnabled {
```

Comment:
nit: should this not also be an instance of `MessengerInterface`

Reply:
Great catch thanks